pkg/controller/jobframework/tas_validation.go

```diff
@@ -82,10 +82,18 @@ func ValidateTASPodSetRequest(replicaPath *field.Path, replicaMetadata *metav1.O
 			allErrs = append(allErrs, field.Required(annotationsPath.Key(kueuebeta.PodSetSliceSizeAnnotation), "slice size is required if slice topology is requested"))
 		}
 	}
-
 	if !sliceRequiredFound && sliceSizeFound {
 		allErrs = append(allErrs, field.Forbidden(annotationsPath.Key(kueuebeta.PodSetSliceSizeAnnotation), fmt.Sprintf("cannot be set when '%s' is not present", kueuebeta.PodSetSliceRequiredTopologyAnnotation)))
 	}
+	if podSetGroupNameFound {
+		if sliceSizeFound {
+			allErrs = append(allErrs, field.Forbidden(annotationsPath.Key(kueuebeta.PodSetSliceSizeAnnotation), fmt.Sprintf("cannot be set when '%s' is present", kueuebeta.PodSetGroupName)))
```

Comment:
```suggestion
			allErrs = append(allErrs, field.Forbidden(annotationsPath.Key(kueuebeta.PodSetSliceSizeAnnotation), fmt.Sprintf("may not be set when '%s' is present", kueuebeta.PodSetGroupName)))
```

Let's follow the validation error message best-practices.
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#error-messages

Reply:
https://github.com/kubernetes-sigs/kueue/pull/7070/commits/aee53474fb631d79278cc0c020bdfb2648451c2f